src/models/room-summary.ts

```diff
@@ -14,11 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import type { Hero } from "./room.ts";
```

Comment:
feels like `Hero` should probably come and live in this file, to avoid a reference cycle.